modules/setup/library/Setup/Webserver/Apache.php

```diff
@@ -11,9 +11,11 @@
 class Apache extends Webserver
 {
     protected $fpmUri = 'fcgi://127.0.0.1:9000';
```

Comment:
The default here should be `127.0.0.1:9000` as well.

Reply:
Why? Apache requires `fgci` here because else it does not know which protocol handler to use.

Reply:
Either, with or without schema, but it is not consistent currently :)

Reply:
True. @dnsmichi Please remove `fcgi://` here. I'll comment some other lines.